shell/client/admin/feature-key-client.js

```diff
@@ -172,6 +181,21 @@ Template.adminFeatureKeyModifyForm.events({
   "click button.feature-key-delete-button"(evt) {
     Template.instance().showForm.set("delete");
   },
+
+  "click .feature-key-renewal-problem button.retry"(evt) {
+    const instance = Template.instance();
+    const state = Iron.controller().state;
+    const token = state.get("token");
+    const renewInFlight = instance.renewInFlight;
+    renewInFlight.set(true);
+    Meteor.call("renewFeatureKey", token, (err) => {
+      renewInFlight.set(false);
+      if (err) {
+        // Note: Renewal failures aren't reported this way. If we get here there was a bug.
+        console.error(err);
```

Comment:
Consider adding better error reporting here (even if it's just an `alert`). Most users won't think to look in the console when an unexpected error occurs, and may even think it succeeded.